compilers/camk.in

```diff
@@ -9,5 +9,5 @@ ifeq ($(PIERNIK_DEBUG), 1)
    F90FLAGS += -ggdb -O0 -ffpe-trap=zero,overflow,invalid -fcheck=all -fno-omit-frame-pointer
    F90FLAGS += -Wall -W -Wextra -pedantic-errors -Wno-unused-function
 else
-   F90FLAGS += -O2 -fstack-arrays
```

Comment:
The difference in performance seems to be of order of 10-20% and happens mostly within RTVD solver (see attached pictures). The crashes with `-fstack-arrays` aren't random - they occur for large local domains and can be avoided by splitting local domain into smaller pieces via AMR_bsize parameter (in which case there is much bigger performance drop and many more arrays are created). IMO it is worth investigating where this difference come from and eliminate it, if possible.
![stack_arrays_1](https://cloud.githubusercontent.com/assets/4697372/26140326/68ab8c06-3ad6-11e7-8380-368f8da16af3.png)
![stack_arrays_2](https://cloud.githubusercontent.com/assets/4697372/26140327/68af8e50-3ad6-11e7-9aa4-12cea6c91215.png)